SmartDeviceLink/SDLScreenshotViewController.m

```diff
@@ -30,18 +32,20 @@ - (instancetype)init {
     return self;
 }
 
-// HAX: https://github.com/smartdevicelink/sdl_ios/issues/1250
+    // HAX: https://github.com/smartdevicelink/sdl_ios/issues/1250
 - (UIInterfaceOrientationMask)supportedInterfaceOrientations {
     UIViewController *viewController = [self sdl_topMostControllerForWindow:[UIApplication sharedApplication].windows[0]];
 
-    if (viewController != nil) {
+    if (viewController == self) {
+        @throw [NSException sdl_invalidLockscreenSetupException];
```

Comment:
Why not just assert and return a default value? Isn't the orientation only an issue for navigation apps using `CarWindow`. Seems kind of pointless to crash a developer's non-navigation app for api that doesn't affect their app.